clients/ts/FunctionalTests/FunctionalTests.csproj

```diff
@@ -44,6 +45,7 @@
     <TypeScriptCompile Include="ts\ConnectionTests.ts" />
     <TypeScriptCompile Include="ts\HubConnectionTests.ts" />
     <TypeScriptCompile Include="ts\index.ts" />
+    <TypeScriptCompile Include="ts\TestLogger.ts" />
```

Comment:
Looks like these ended up getting checked in. TBH I feel like we should leave them in as long as VS is going to keep insisting on creating them. Once VS is fixed, we can remove them.